src/averageAges.js

```diff
@@ -20,6 +20,16 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = people.filter((person) => {
+    return century
+      ? ((Math.ceil(person.died / 100) === century) && person.sex === 'm')
+      : (person.sex === 'm');
+  })
+    .map(age => age.died - age.born);
+
+  let meanAge = men.reduce((sum, age) => sum + age, 0);
+
+  return Math.round((meanAge /= men.length) * 100) / 100;
```

Comment:
Why do you use assignment here?
Fix this and other 2 occastions.

```suggestion
  const meanAge = men.reduce((sum, age) => sum + age, 0);

  return Math.round((meanAge / men.length) * 100) / 100;
```